### DISCUSSION TITLE
Remove time limit from Costumes

### DISCUSSION THREAD

**Discussion by Randalftown** [chromiecraft/chromiecraft#7265] [Suggestions]
I don’t understand why costumes have a timer on them, let me pop one on and pop it off when I feel like it like on other servers. I would buy more of them if they were permanent and toggleable. Let me be a Werewoof rouge prowling the forests of the human lands.

**Comment by JimJamJamie** [comment]
Rogue*

Sounds good though.

**Comment by ChaTheWitch** [comment]
Yep. Yes. Let me be the succubus I always wanted to be. A holy light casting succubus demon dommy mommy.
